docs/setup_guides/cloud.rst

```diff
@@ -1,5 +1,5 @@
-Getting Started with Dask on Kubernetes
-=======================================
+Getting Started with Pangeo on the Cloud
```

Comment:
I'm hoping @jacobtomlinson and @mrocklin can work on this setup guide. The text below is currently for dask-kubernetes but it would be good to provide the pangeo-specific details of how to deploy the scalable jupyterhub system we have, referencing zero-to-jupyterhub, and kubernetes documentation whenever possible.

Perhaps @jgerardsimcock and/or @tjcrone have ideas of what details are needed here.

Reply:
@tjcrone recently told me he basically had such a setup guide already written.